File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int ans = 0;
        
        for(int i = 0; i < nums.size(); i++){
            int current = 0;

            while(nums[i]){
                nums[i] /= 10;
                current++;
            }

            if(current % 2 == 0)
                ans++;
        }

        return ans;
    }
};

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int ans = 0;
        int n = nums.length;

        for(int i : nums){
            String number = Integer.toString(i);

            if(number.length() % 2 == 0)
                ans++;
        }

        return ans;
    }
}